unpick-definitions/entity_status.unpick

```diff
@@ -0,0 +1,70 @@
+v2
+
+constant entity_statuses net/minecraft/entity/EntityStatuses SET_OP_LEVEL_3
+constant entity_statuses net/minecraft/entity/EntityStatuses SET_OP_LEVEL_4
+constant entity_statuses net/minecraft/entity/EntityStatuses BLOCK_WITH_SHIELD
+constant entity_statuses net/minecraft/entity/EntityStatuses BREAK_SHIELD
+constant entity_statuses net/minecraft/entity/EntityStatuses PULL_HOOKED_ENTITY
+constant entity_statuses net/minecraft/entity/EntityStatuses HIT_ARMOR_STAND
+constant entity_statuses net/minecraft/entity/EntityStatuses DAMAGE_FROM_THORNS
+constant entity_statuses net/minecraft/entity/EntityStatuses STOP_LOOKING_AT_VILLAGER
+constant entity_statuses net/minecraft/entity/EntityStatuses USE_TOTEM_OF_UNDYING
+constant entity_statuses net/minecraft/entity/EntityStatuses DAMAGE_FROM_DROWNING
+constant entity_statuses net/minecraft/entity/EntityStatuses DAMAGE_FROM_FIRE
+constant entity_statuses net/minecraft/entity/EntityStatuses ADD_DOLPHIN_HAPPY_VILLAGER_PARTICLES
+constant entity_statuses net/minecraft/entity/EntityStatuses STUN_RAVENGER
+constant entity_statuses net/minecraft/entity/EntityStatuses TAME_OCELOT_FAILED
+constant entity_statuses net/minecraft/entity/EntityStatuses TAME_OCELOT_SUCCESS
+constant entity_statuses net/minecraft/entity/EntityStatuses ADD_SPLASH_PARTICLES
+constant entity_statuses net/minecraft/entity/EntityStatuses ADD_CLOUD_PARTICLES
+constant entity_statuses net/minecraft/entity/EntityStatuses DAMAGE_FROM_BERRY_BUSH
+constant entity_statuses net/minecraft/entity/EntityStatuses CREATE_EATING_PARTICLES
+constant entity_statuses net/minecraft/entity/EntityStatuses ADD_PORTAL_PARTICLES
+constant entity_statuses net/minecraft/entity/EntityStatuses BREAK_MAINHAND
+constant entity_statuses net/minecraft/entity/EntityStatuses BREAK_OFFHAND
+constant entity_statuses net/minecraft/entity/EntityStatuses BREAK_HEAD
+constant entity_statuses net/minecraft/entity/EntityStatuses BREAK_CHEST
+constant entity_statuses net/minecraft/entity/EntityStatuses BREAK_LEGS
+constant entity_statuses net/minecraft/entity/EntityStatuses BREAK_FEET
+constant entity_statuses net/minecraft/entity/EntityStatuses ADD_SPRINTING_PARTICLES_OR_RESET_SPAWNER_MINECART_SPAWN_DELAY
```

Comment:
huh

Reply:
Imo this is good. We don't know if they could have a separate field for this minecart delay, but having this here imo is good enough

Reply:
There is. its `field_30951`, but it is `private static final`.

Reply:
I think its fine if the constants file is a little messy if it covers all of the cases. I don't think its going to confuse too many people. also mojang should have things overlap, so we work with what we have

Reply:
>  but it is `private static final`.

Hmm, is there a public alternative then? Something must be using these values from fields elsewhere.

Reply:
no not that i saw. its only referenced once in `MobSpawnerLogic#handleStatus`

Reply:
Hmm, in fact this is the 1st field in the class and has value `1`. Dunno how stitch assign number to the fields; it is just weird.